Sources/GeneratorCLI/GeneratorCLI.swift

```diff
@@ -98,7 +98,13 @@ struct GeneratorCLI: AsyncParsableCommand {
         shouldUseDocker: self.withDocker,
         isVerbose: self.verbose
       )
-      .generateBundle(shouldGenerateFromScratch: !self.incremental)
+      do {
+        try await generator.generateBundle(shouldGenerateFromScratch: !self.incremental)
+        try await generator.shutDown()
+      } catch {
+        try await generator.shutDown()
+        throw error
+      }
```

Comment:
It would be nice add a `withGenerator` wrapper which could take care of shutting down and rethrowing if necessary.